modules/obj/test/data/cube_vertex_colors.obj

```diff
@@ -0,0 +1,24 @@
+# Blender 3.6.0
```

Comment:
Nit: Prefer that we use dashes instead of underscores in the file name, to stay as consistent as possible